piety-promoted-v3/modernized/13-chapter-13.adoc

```diff
@@ -224,13 +223,13 @@ and took but little nourishment.
 Towards evening, a gentle slumber afforded some respite;
 but afterwards her cough returned, her breathing became increasingly difficult,
 and her bodily conflict was great.
-Some of her friends u:ho had retired, came again to her chamber.
+Some of her friends who had retired, came again to her chamber.
 She appeared glad to see them, and with earnestness said, "`Rejoice evermore,
 and in everything give thanks.`"
 Thankfulness, no doubt, reigned in her soul,
 in the expectation of a speedy relief from the oppressive state of her enfeebled frame.
 But this she yet was permitted to endure for a time.`'
-This, '`she said, '`is a hard conflict.
+This, '`she said,`' is a hard conflict.
```

Comment:
I think we've got the wrong thing wrapped in single quotes here, I think everything but `she said,` should be quoted.

```ASCIIDOC
'`This,`' she said, '`is a hard conflict.
```

Reply:
haha, it's funny how I mess these up sometimes. I think these are the times there's too much noise in the house. 😄